src/pages/gd/Claim/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react'
+import Title from 'components/gd/Title'
+import { t } from '@lingui/macro'
+import ClaimButton from './ClaimButton'
+import { useLingui } from '@lingui/react'
+import { PageContent, StyledLayout } from "./styled";
+
+const Claim = () => {
+	const { i18n } = useLingui()
+
+	return (
+		<StyledLayout classes="md:mt-24 xl:mt-0 sh:mt-30">
```

Comment:
use NativeBase's layouts: read https://docs.nativebase.io/box and sections below